src/main/java/com/gluonhq/substrate/target/PosixTargetConfiguration.java

```diff
@@ -46,4 +54,37 @@ void checkPlatformSpecificClibs(Path clibPath) throws IOException {
         if (!Files.exists(libjvmPath)) throw new IOException("Missing library libjvm.a not in linkpath "+clibPath);
     }
 
+    /**
+     * Prevent undesired dialogs and build failures when running on MacOs 1.15.0+.
```

Comment:
similar here. If this is only applicable to Mac, it doesn't belong in Posix

Reply:
Same